common/models/service-process.json

```diff
@@ -44,6 +44,11 @@
       "default": 0,
       "description": "Watchdog timeout, in milliseconds. (0 = watchdog disabled)"
     },
+    "watchdogStallout": {
```

Comment:
Isn't this basically the same as `ProfileData.coun({where: {ServiceProcessId: pid}})` (or something like that).